onnxruntime/python/tools/quantization/README.md

```diff
@@ -25,36 +22,45 @@ The following ops were chosen as phase 1 ops because in most of the CNN models t
 
  Zero point represents zero in quantization space. It is important that floating point zero value be exactly representable in quantization space. This is because in lot of CNNs, zero padding is used and if after quantization it is not possible to represent 0 uniquely then it will lead to accuracy errors.
 
-## Quantization and model opset versions
+## Quantizing an onnx model
+There are 2 ways of quantizing a model
+
+* Only use quantization : This method assumes the model owner is going to use Integer Ops for quantization or has pre calculated the quantization params as they are required inputs for using QLinear Ops
+
+ONNX Model ---> quantize.py ---> ONNX Quantized Model
+
+* Use both calibration and quantization : This method is preferred when using QLinear Ops for quantization.
+
+ONNX Mode --> calibrate.py --> quantize.py --> ONNX Quantized model
+
+Today ORT does not guarantee support for E2E model quantization, meaning since not all ONNX ops have support for 8 bit data types therefore only the supported ops in the model are quantized. For rest of the ops inputs are reconverted to FP32.
+
+### List of Supported Quantized Ops:
+The following ops were chosen as phase 1 ops because in most of the CNN models these ops consume most amount of compute and power and therefore there is benefit in quantizing these ops to get perf benefits.
+ * Convolution
+ * Matmul
+ * Data type agnostic ops like transpose, identity etc ( Note: special quantization is not done for these ops.)
+
```

Comment:
i think i saw special treatment of Gather as well since the spec supports int8? 
is it worth a mention?

Reply:
Gather is a data type agnostic op... special quantization is not required for this op and It was added for vscode. The next step is to change this and support all the data type agnostic ops. I would defer the mention of this until this support it added. Just explaining why gather was added can be confusing. 

The idea is - if we come across data type agnostic ops which are touched during quantization i.e. they share weights\inputs with an op which is being quantized or they are in between 2 ops which are being quantized then the input to these ops will not be dequantzed to FP32. This will save dequantization+quantization step around these ops.

Reply:
Gather is important since quantized Bert model usually has embedding quantized to 8bit, and embedding weights are input to Gather node. Of course, this could be a future work to support embedding quantization.